packaging/buildTasks.mjs

```diff
@@ -34,32 +34,38 @@ async function readJson(filePath) {
     return JSON.parse((await readFile(filePath)).toString())
 }
 
-export async function packContractsAndChallenges() {
-    const resources = join("resources", "challenges")
-
+async function createResourcesFolder(resources) {
     if (!existsSync(resources)) {
         await mkdir(resources)
     } else {
         for (const crp of await readdir(resources)) {
             await unlink(join(resources, crp))
         }
     }
+}
+
+async function handleFile(resources, name, contents) {
+    const targetName = createHash("md5").update(name).digest("hex")
+
+    await writeFile(
+        join(resources, `${targetName}.crp`),
```

Comment:
CRP = challenge resource package, this is a new format, and it doesn't really make sense to have it as the same extension if it stores different data.

Reply:
See my comment on Anthony's response to that.